policy/modules/roles/sysadm.te

```diff
@@ -100,6 +100,10 @@ ifdef(`init_systemd',`
 	systemd_dbus_chat_networkd(sysadm_t)
 	fs_read_nsfs_files(sysadm_t)
 
+	systemd_run_nspawn(sysadm_t, sysadm_r)
+	systemd_run_passwd_agent(sysadm_t, sysadm_r)
+	systemd_watch_passwd_runtime_dirs(sysadm_t)
```

Comment:
Are these related to running nspawn interactively? If so they should go in the `systemd_run_nspawn()` interface.